templates/partials/financial-summary.html

```diff
@@ -6,7 +6,7 @@ <h2 id="section-1-header" tabindex="0">Financial Summary</h2>
       </div>
 
       {% if primary_committee or authorized_committees %}
-      <div class="page-subsection">
+      <div class="page-subsection page-subsection--noline">
```

Comment:
I actually want the line here between the authorized committee section and the joint fundraising committees. Fine to keep the css class, but let's remove it from here.